README.md

```diff
@@ -1,31 +1,44 @@
 # Design System Starter
+![verify and publish workflow](https://github.com/Zuehlke/design-system-starter/actions/workflows/verify-and-publish.yml/badge.svg)
+
+The **Design System Starter** provides a solid foundation for creating your own custom design system. This starter kit 
+aims to streamline the process of building consistent and visually appealing web components for your design system.
+
+## Key Features
+
+### 🚀 Design Token Integration
```

Comment:
Maybe mention here that we just need a json in the style-dictionary format. So whatever tool that can export a json with a supported format is going to work with our tool.
But ofc we can say, our know it works with this plugin and it is the only officially supported.
Because afaik, Lucas and Co do not even use the plugin, they just change it in the json itself ^^

Reply:
okk you went into more depth in the design-tokens.md
not sure if we want to mention here the plugin (i think this is soon not needed anymore)

Reply:
For now, let's wait and see :+1: